src/components/Lend/LoanSearch/LoanSearchFilter.vue

```diff
@@ -93,6 +99,20 @@
 					</h2>
 				</button>
 			</kv-accordion-item>
+			<kv-accordion-item id="acc-distribution-model" :open="false">
```

Comment:
Please move this above the Advanced Filters section outside of that section. My apologies for the bad spec.

This will need to be gated by the experiment Setting there as well.

Reply:
Should it remain in an accordion like in the screenshot? It's currently in an accordion just below the advanced filters accordion. If it's outside an accordion, does it need a header like in legacy? The gender filter doesn't have a header, but I could see these options being more confusing without a header.

Reply:
This should be an independent section without an accordian. It should have a header that matches legacy lend.

Reply:
Header and sub header? We have taken some liberties in transferring filters to the new UI, for example legacy sector does have a header, where in the new UI there isn't. I think we need to get the specifics into the tickets to reduce churn on these PRs.

![image](https://user-images.githubusercontent.com/16867161/195435152-30cdaa8f-a912-4091-a977-bd64cc7f8db2.png)

Reply:
Good point. Let's leave off the sub head for now and spend some time refining the other tickets before getting started on them.

Reply:
Done. Reused capitalized header styling for consistency with filter weight of importance.

![image](https://user-images.githubusercontent.com/16867161/195437058-a9b17798-12bc-4ed0-920c-e3564db3239b.png)